connman/plugins/clat.c

```diff
@@ -1350,15 +1435,59 @@ static char *cidr_to_str(unsigned char cidr_netmask)
 	return g_strdup(netmask);
 }
 
+static int setup_ipv4_default_route(struct clat_data *data, bool enable)
+{
+	int index;
+	int err;
+
+	if (!clat_settings.clat_device_setup_default_route) {
+		DBG("disabled by config");
+		goto out;
```

Comment:
Yes, since the value below (`ipv4_default_route_on`) needs to be changed. The value is used elsewhere for double nat and restart cases. Well, not the cleanest way of doing things but the option itself was merely a failsafe for testers if the current approach of letting ConnMan setup the default route did not work. There are future tasks for this and it might be good to do cleanup there with fresh eyes.